frontend/src/scenes/session-recordings/replayPaletteCommands.ts

```diff
@@ -0,0 +1,100 @@
+import { IconRewindPlay } from '@posthog/icons'
+import { Command, GLOBAL_COMMAND_SCOPE } from 'lib/components/CommandPalette/commandPaletteLogic'
+import { isURL } from 'lib/utils'
+import { urls } from 'scenes/urls'
+
+import {
+    FilterLogicalOperator,
+    PropertyFilterType,
+    PropertyOperator,
+    RecordingDurationFilter,
+    RecordingUniversalFilters,
+    ReplayTabs,
+} from '~/types'
+
+function isUUIDLike(candidate: string): boolean {
+    return candidate.length === 36 && !!candidate.toLowerCase().match(/^[0-9a-f-]+$/)?.length
```

Comment:
we don't have a UUID type right? otherwise, we could just `try UUID(candidate) catch ...`